### PR TITLE
feat : extension screens & weather api

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR
- Adding weather API from [openweathermap](https://openweathermap.org/)
- Adding greeting green that asks user's name and focus of the day
- Adding the main screen that display's a user name, focus, time, weather

Review these files : [Click here](https://github.com/sudipkundu999/cobra-chrome-extension/pull/1/files?file-filters%5B%5D=.html&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-deleted-files=true&show-viewed-files=true)
Also, **please ignore quotesDB.js** -> it consists of quotes ( 25275 lines )

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
1.Amazing work how you handle all the API 
2. How you add the local storage 
3.flow of code 
Looks good to merge

**Review by DivyanshSareen — APPROVED** [review]
Code looks clean, neat weather api calls.

File: src/components/Greeting.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export const Greeting = () => {
+  const [greeting, setGreeting] = useState("");
+
+  useEffect(() => {
+    const greetTime = new Date().getHours();
+    if (greetTime < 4) setGreeting("Good Night");
+    else if (greetTime < 11) setGreeting("Good Morning");
+    else if (greetTime < 16) setGreeting("Good Afternoon");
+    else if (greetTime < 19) setGreeting("Good Evening");
+    else setGreeting("Good Night");
```
Comment (DivyanshSareen) [inline src/components/Greeting.jsx:12]: Can use switch here

Comment (sudipkundu999) [reply src/components/Greeting.jsx:12]: I was considering switch-case first but then the lines of code increased, so I decided to stick with the if-else ladder.

**Pull request merged**